homeassistant/components/tuya/light.py

```diff
@@ -441,9 +444,32 @@ def is_on(self) -> bool:
 
     def turn_on(self, **kwargs: Any) -> None:
         """Turn on or control the light."""
+        _LOGGER.debug("light kwargs -> %s, cm = %s", kwargs, self.color_mode)
```

Comment:
This line has been removed in previous versions, it's not needed.

If you need to debug on this level, use a debugger.

```suggestion
```